projects/power_selection/src/power_selection.c

```diff
@@ -36,8 +37,12 @@ static uint16_t s_aux_temp = 0;
 uint16_t prv_status_checker() {
   adc_read_raw(aux_channels[AUX_ADC_VOLT_CHANNEL], &s_aux_volt);
   adc_read_raw(aux_channels[AUX_ADC_TEMP_CHANNEL], &s_aux_temp);
-  // LOG_DEBUG("AUX Volatge Data: %d\n", aux_volt);
-  // LOG_DEBUG("AUX Temp Data: %d\n", aux_temp);
+  LOG_DEBUG("AUX Volatge Data: %d\n", s_aux_volt);
+  LOG_DEBUG("AUX Temp Voltage Data: %d\n", s_aux_temp);
+  double resistance = 33000.0 / (double)(s_aux_temp / 1000.0) - 10000;
```

Comment:
It's not great to leave this here since you'll need it for tests, you could add the calculation as a #define in the header that takes in a parameter, e.g.
```
#define temp_to_res(r) 33000.0 / (double)((r) / 1000.0) - 10000
```
This way you can reuse it exactly in test.